odysseus/halow_setup/README.md

```diff
@@ -0,0 +1,187 @@
+This is the complete process to get Halow wifi working.
```

Comment:
Not sure if you're looking for comments on this as it is mostly for you rn but we should probably generalize this and specify what is needed to initialize. I.e. does the Pi need an internet connection, does the host system need to be Linux, etc. Imo making sure this is standardized for a stock ubuntu build is good since that is probably what most newbies in linux will default to i.e. this entire club lmao

Reply:
Alright, will do.  Making it work on ubuntu might present some challenges, so I will add that to the PR last.

Reply:
Just to clarify, I mean like Ubuntu on the host PC side, not Ubuntu on the Pi lol

Reply:
Oh ok ya thats much easier I got scared for a second haha.